python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -82,8 +107,11 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
         logger.info("Initializing")
         Ganga.Runtime._prog.initEnvironment()
     else:
-        from Ganga.Runtime.Repository_runtime import startUpRegistries
+        # Load the config from the appropriate files
+        load_config_files()
```

Comment:
If this can be pulled out why isn't it just pulled out as a method in the Ganga code then and called from here. I don't see any real benefit in duplicating functionality of 'read the config system' in 2 or more places. If we ever change 1 we run the risk that the other becomes out of sync for no good reason.